mariadb/content.md

```diff
@@ -12,38 +12,48 @@ The %%IMAGE%% has a number of tags, and of note is `latest`, as the latest stabl
 
 ## Running the container
 
+### Configuration
+
+#### Port binding
+
+By default, the database running within the container will listen on port 3306. You can expose the container port 3306 to the host port 3306 with the `-p 3306:3306` argument to `docker run`, like the command below:
+
+```console
+$ docker run --name some-%%REPO%% -p 3306:3306 %%IMAGE%%:latest
+```
+
 ### Starting using a minimal configuration
 
 The environment variables required to use this image involves the setting of the root user password:
 
 ```console
-$ docker run --detach --name some-%%REPO%% --env MARIADB_ROOT_PASSWORD=my-secret-pw  %%IMAGE%%:latest
+$ docker run --detach --name some-%%REPO%% --env MARIADB_ROOT_PASSWORD=my-secret-pw %%IMAGE%%:latest
 ```
 
 or:
 
 ```console
-$ docker run --detach --name some-%%REPO%% --env MARIADB_ALLOW_EMPTY_ROOT_PASSWORD=1  %%IMAGE%%:latest
+$ docker run --detach --name some-%%REPO%% --env MARIADB_ALLOW_EMPTY_ROOT_PASSWORD=1 %%IMAGE%%:latest
 ```
 
 or:
 
 ```console
-$ docker run --detach --name some-%%REPO%% --env MARIADB_RANDOM_ROOT_PASSWORD=1  %%IMAGE%%:latest
+$ docker run --detach --name some-%%REPO%% --env MARIADB_RANDOM_ROOT_PASSWORD=1 %%IMAGE%%:latest
 ```
 
 ... where the container logs will contain the generated root password.
 
-## %%STACK%%
+## %%Docker Compose%%
```

Comment:
This has to be `%%STACK%%` for the templating to replace it with the content of [`.template-helpers/stack.md`](https://github.com/docker-library/docs/blob/c8fefb5ed65cec4609a4cc64b1154ffeb240d2fa/.template-helpers/stack.md) (inserting in `stack.yml` as well). If the content of `stack.md` is not correct, then the file would have to be copied into the `mariadb/` folder and modified.

Though looking at `update.sh`, we can just rename `mariadb/stack.yml` to `mariadb/docker-compose.yml` and change this to `%%COMPOSE%%` to get the older [compose template](https://github.com/docker-library/docs/blob/c8fefb5ed65cec4609a4cc64b1154ffeb240d2fa/.template-helpers/compose.md).